### DISCUSSION TITLE
App translations

### DISCUSSION THREAD

**Discussion by malliaridis** [malliaridis/obssd#27] [General]
## Idea description
The app could be translated in various languages so native speakers will have a translated app instead of the default english language.

Currently requested and supported languages:
* ✔ English
* ✔ German

Write in the comments below if you want more languages.

**Comment by enriquespsa** [comment]
It would be a great idea to add the Spanish language.

**Reply by malliaridis** [reply]
Thanks for your request. Spanish is indeed the second most frequently used language in the app (after english), but it is not supported yet. I was planning in adding the language, but due to the texts in the "Help" section I was not be able translate those properly.

However, with the next update the help section will very likely change and that will allow me to reuse most translations of the current OBS Studio translations.

@EnriqueJuegos is there any way you could support me with correction-reading once that is available?
